src/components/PreferencesBody.vue

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+import Vue from 'vue';
+import PreferencesBodyApplication from '@/components/PreferencesBodyApplication.vue';
+import PreferencesBodyVirtualMachine from '@/components/PreferencesBodyVirtualMachine.vue';
+import PreferencesBodyContainerRuntime from '@/components/PreferencesBodyContainerRuntime.vue';
+import PreferencesBodyKubernetes from '@/components/PreferencesBodyKubernetes.vue';
+
+export default Vue.extend({
```

Comment:
Perhaps I can revisit because I didn't anticipate this pattern to change.. my plan was to manage preferences centrally within a Preferences page and treat each component represented here more like a tabbed interface that renders data and emits changes back up the component tree.

Reply:
Hmm, I guess that should be fine, just means slightly more code?
I was assuming a vuex store for the (uncommitted) settings, but either one is fine.